scripts/benchmark_migration.py

```diff
@@ -160,7 +160,6 @@ def main(
         rows = session.query(model).count()
         print(f"- {model.__name__} ({rows} rows in table {model.__tablename__})")
         model_rows[model] = rows
-    session.close()
```

Comment:
Not sure why we were closing the session here, but it's not needed and actually could cause problems.